Buildings/Fluid/HeatExchangers/ThermalWheels/Data/Generic.mo

```diff
@@ -86,12 +119,12 @@ corrections versus wheel speed ratio are correction factors that are multiplied
 with the heat exchange effectiveness that the wheel has a full rotational speed.
 </li>
 <li>
-When <code>haveLatentHeatExchange = true</code>,
+When <code>haveLatentHeatExchange=true</code>,
```

Comment:
Has the flag been used to enable/disable parameters? I didn't see it.

Reply:
no, it was not used. I fixed this issue.